README.md

```diff
@@ -89,6 +104,12 @@ sentence-level embeddings.</summary>
 > The goal of MatchZoo is to provide a high-quality codebase for deep text matching research, such as document retrieval, question answering, conversational response ranking, and paraphrase identification.
 </details>
 
+<details>
+<summary><a href="https://github.com/flairNLP/flair">Flair</a> - An NLP library with state-of-the-art sentence and document embeddings for novel use cases</summary>
+
+> It offers a novel method named TARS, and pretrained models in several languages, for zero-shot named entity recognition beside zero-shot text classification, and semantic search.
+</details>
```

Comment:
Flair looks like a great NLP library, but I am not sure if it fits the topic of metric learning.
Document embedding tutorial they are providing is rather basic https://github.com/flairNLP/flair/blob/master/resources/docs/TUTORIAL_9_TRAINING_LM_EMBEDDINGS.md 
No significant difference with what huggingface does out of the box

I suggest to either remove it for now, or provide more specific links to exact use-cases of Flair in metric learning